pkg/apis/core/validation/validation.go

```diff
@@ -5170,6 +5170,11 @@ func ValidatePodUpdate(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 		allErrs = append(allErrs, field.Invalid(fldPath, newPod.Status.QOSClass, "Pod QoS is immutable"))
 	}
 
+	isPodResizeRequestValid := isPodResizeRequestValid(*oldPod)
+	if !isPodResizeRequestValid {
+		allErrs = append(allErrs, field.Forbidden(specPath, "Pod running on node without IPPVS enabled may not be updated"))
+	}
```

Comment:
This will probably be OK once we switch to the resize subresource (https://github.com/kubernetes/kubernetes/issues/109553), but to call it here you'll need to confirm it's actually a resize request (there are other valid pod updates, including metadata & image)

Reply:
@tallclair In order to test this, I created a new branch from Anish's PR and added these changes to the new `ValidatePodResize` function that is added. This way this validation error would be added only to resize requests and all the other tests wouldn't fail like in this PR. Once Anish's PR gets merged I can move these changes to the `ValidatePodResize` function too. I opened https://github.com/kubernetes/kubernetes/pull/128540 to test this first and be ready before code freeze. I'll close it once I'm done testing.

Reply:
Yes, agreed - once Anish's PR merges this should just move to `ValidatePodResize`, and you don't need to worry about the feature gate.